packages/ui/src/proposals/components/ProposalList/ProposalListItem.tsx

```diff
@@ -74,24 +79,38 @@ export const ProposalListItem = ({ proposal, isPast, memberId, isCouncilMember }
         </Tooltip>
       </StageField>
       <MemberInfo member={proposal.proposer} memberSize="s" showIdOrText />
-      {isPast && (
-        <StageField>
-          <Subscription>{toDDMMYY(displayDate)}</Subscription>
-        </StageField>
-      )}
+      <StageField>{isPast && <Subscription>{toDDMMYY(displayDate)}</Subscription>}</StageField>
       <StageField>
         <ProposalItemVoteDetails proposal={proposal} memberId={memberId} isCouncilMember={isCouncilMember} />
       </StageField>
+      <CopyButton
+        square
+        size="small"
+        textToCopy={getUrl({ route: ProposalsRoutes.preview, params: { id: proposal.id } })}
+        icon={<LinkIcon />}
+      />
     </ProposalItem>
   )
 }
 
+const CopyButton = styled(CopyButtonTemplate)`
+  justify-self: end;
+  visibility: hidden;
+  transition: none;
+  & svg {
+    transition: none !important;
```

Comment:
@thesan any idea to avoid this `!important`?

Reply:
@freakstatic these `transition: all ...` are becoming annoying and I don't even know which properties they should target, so rewriting them properly will be difficult :unamused: .

Thankfully in this case since the button is at the end of the row, replacing `visibility: hidden`/`visibility: visible` by `display: none`/`display: flex` should work without overriding the transitions

Reply:
yeah that works but the table does a strange effect when dragging

Reply:
![image](https://user-images.githubusercontent.com/7486476/171968812-5cc3aa5f-c34d-414b-811c-2ac507d015ff.png)
![image](https://user-images.githubusercontent.com/7486476/171968846-d81c772c-e7c3-4dd6-bdf8-22b976ad13d3.png)

Reply:
not sure you can notice in the two images, but the table grows a bit to the right side on mouse hover